### PR TITLE
both filter now work

### PR DESCRIPTION
**Pull request by matthias-luger**

### CODE REVIEW

**Review by Ekwav — CHANGES_REQUESTED** [review]
please add a cypress test to make sure this doesn't occur again.

**Comment by matthias-luger** [comment]
> please add a cypress test to make sure this doesn't occur again.

i beg you not

File: src/components/BidList/BidList.tsx [hunk 1]
```diff
@@ -90,6 +90,10 @@ function BidList(props: Props) {
             })
             setBids(bids);
 
+            if(bids.length < 10 && newBids.length !== 0){
```
Comment (Ekwav) [inline src/components/BidList/BidList.tsx:93]: This change is redundant, please share code with AuctionList.tsx

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: src/components/BidList/BidList.tsx (hunk 1 above)
Comment (matthias-luger) [reply src/components/BidList/BidList.tsx:93]: historisch bedingt

**Review by Ekwav — APPROVED** [review]
looks ok

**Pull request merged**